.github/scripts/check-anchors.py

```
@@ -32,15 +32,21 @@ def slugify(text: str) -> str:
     # escaped underscore in markdown source -> hyphen in slug
     text = text.replace('\\_', '-')
     text = text.lower()
-    # normalize unicode so e.g. full-width chars compare sanely
-    text = unicodedata.normalize('NFKC', text)
+    # Mintlify keeps full-width CJK punctuation (（）、。) in anchors, e.g.
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)